src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -728,9 +728,7 @@
       (schema/optional-key "facts") {(schema/required-key "values") {schema/Str schema/Any}}
       (schema/optional-key "transaction_uuid") schema/Str
       (schema/optional-key "environment") schema/Str
-      (schema/optional-key "options") {(schema/optional-key "capture_logs") schema/Bool}
```

Comment:
I think we want to keep these options.

Reply:
We do, they're below. GH is just showing it as removed because there's an additional curly brace on this line now.

Reply:
Well now there's a conflict, prolly because the node change introduced a new key for the options map.

Reply:
:fistshake: